docs/running_on_mainnet.md

```diff
@@ -1 +1,121 @@
-# TODO(const)
\ No newline at end of file
+# Running on the Main Network
+This tutorial shows how to use the bittensor `btcli` to create a subnetwork and connect your mechanism to it. It is highly recommended that you run `running_on_staging` and `running_on_testnet` first before attempting to register on mainnet. Mechanisms running on the main are open to anyone, emit real TAO, and incurs a `lock_cost` in TAO to create, and you should be careful not to expose your private keys, to only use your testnet wallet, not use the same passwords as your mainnet wallet, and make sure your mechanism is resistant to abuse. 
+
+
+## Steps
+
+1. Clone and Install Bittensor and the Bittensor Subnet Template.
+This clones and installs the template if you dont already have it (if you do, skip this step)
+```bash
+cd .. # back out of the subtensor repo
+git clone https://github.com/opentensor/bittensor-subnet-template.git # Clone the bittensor-subnet-template repo
+cd bittensor-subnet-template # Enter the bittensor-subnet-template repo
+python -m pip install -e . # Install the bittensor-subnet-template package
+```
+
+2. Create wallets for your subnet owner, for your validator and for your miner.
+This creates local coldkey and hotkey pairs for your 3 identities. The owner will create and control the subnet and must have at least 100 TAO on it before it can run step 3. The validator and miner will run the respective validator/miner scripts and be registered to the subnetwork created by the owner.
+```bash
+# Create a coldkey for your owner wallet.
+btcli wallet new_coldkey --wallet.name owner
+
+# Create a coldkey and hotkey for your miner wallet.
+btcli wallet new_coldkey --wallet.name miner
+btcli wallet new_hotkey --wallet.name miner --wallet.hotkey default
+
+# Create a coldkey and hotkey for your validator wallet.
+btcli wallet new_coldkey --wallet.name validator
+btcli wallet new_hotkey --wallet.name validator --wallet.hotkey default
+```
+
+3. Getting the price of subnetwork creation
+Creating subnetworks on mainnet is competitive and the cost it determined by the rate at which new network are being registered onto the chain. By default you must have at least 100 TAO on your owner wallet to create a subnetwork. However the exact amount will fluctuate based on demand. The code below shows how to get the current price of creating a subnetwork.
+```bash
+btcli subnet lock_cost --subtensor.network finney
```

Comment:
I believe that `--subtensor.network finney` is the default network. If so, we can remove.